### ISSUE TITLE
Error in bootstrap script

### ISSUE THREAD

**Issue by nweyand** [qTox/qTox#4302]
wget seems to be unable to retrieve some dependency:

```
[...]
SYSTEM_WGETRC = c:/progra~1/wget/etc/wgetrc
syswgetrc = c:\Program Files (x86)\GnuWin32/etc/wgetrc
--2017-03-28 04:27:59--  http://fukuchi.org/works/qrencode/qrencode-3.4.4.tar.gz
Resolving fukuchi.org... 173.230.148.37
Connecting to fukuchi.org|173.230.148.37|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://fukuchi.org/works/qrencode/qrencode-3.4.4.tar.gz [following]
--2017-03-28 04:28:00--  https://fukuchi.org/works/qrencode/qrencode-3.4.4.tar.gz
Connecting to fukuchi.org|173.230.148.37|:443... connected.
ERROR: cannot verify fukuchi.org's certificate, issued by `/C=US/O=GeoTrust Inc./CN=RapidSSL SHA256 CA':
  Self-signed certificate encountered.
ERROR: certificate common name `fukuchilab.org' doesn't match requested host name `fukuchi.org'.
To connect to fukuchi.org insecurely, use `--no-check-certificate'.
Unable to establish SSL connection.
tar: qrencode-3.4.4.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
[...]
```

**Comment by sudden6** [comment]
That's not an error in the script, the page where it downloads a dependency uses a wrong SSL certificate.

**Comment by nweyand** [comment]
Yeah, I know that. But for the purpose of oversimplification: If the script doesn’t work as it is, it needs fixing. One cannot provide a script which doesn’t run error free and then say “We’re sorry it doesn’t work – but it’s not our fault!” ;)

**Comment by sudden6** [comment]
Agreed, that was to clarify the issue for whoever is going to fix it.

**Comment by nweyand** [comment]
I am working on fixing it and I’ve also contacted the server admin to inform him about the issue.

**Comment by sudden6** [comment]
I assigned you to make it clear you're working on that problem.

**Comment by nweyand** [comment]
thx

**Comment by nweyand** [comment]
There you go.

Note that this doesn’t fix other existing problems like that some of the libraries are at outdated versions etc. It also doesn’t address the massive amount of warnings generated during the compile process mentioned in #4307.

**Comment by Diadlo** [comment]
Fixed in #4463
